### PR TITLE
Updated install instructions in README

### PR DESCRIPTION
**Pull request by dspreitz**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,6 +27,54 @@ ts-node examples/calc-ranking.ts fixtures/2017-07-17-lev
...
 
 
+Installation
+------------------------------------------------------------------------------
+To install aeroscore you need to install:
+
+Node.js
+```bash
+curl -sL https://deb.nodesource.com/setup_10.x | sudo -E bash -
+sudo apt-get install -y nodejs
```
Comment (Turbo87) [inline README.md]: I'd recommend to install Node using https://github.com/creationix/nvm

File: README.md [hunk 2]
```diff
@@ -27,6 +27,54 @@ ts-node examples/calc-ranking.ts fixtures/2017-07-17-lev
...
+
+Yarn
+```bash
+curl -o- -L https://yarnpkg.com/install.sh | bash
+```
+
+Logout (exit) and back in again
+
+```bash
+yarn global add ts-node
```
Comment (Turbo87) [inline README.md]: you can combine this to `yarn global add ts-node typescript`

File: README.md (hunk 1 above)
Comment (dspreitz) [reply README.md]: OK.

**Comment by Turbo87** [comment]
Merci

**Pull request merged**